lpa-submissions-e2e-tests/cypress/integration/healthAndSafety.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Appeals Extra Conditions
+  As a beta LPA I want to know if there are any health and safety issues at the appeal site
+  So that I take the appropriate precautions
+
+  Background:
+    Given an appeal has been created
+    And a questionnaire has been created
+
+  Scenario: AC01 - User navigates to 'Are there any health and safety issues on the appeal site?' page
+    Given a LPA Planning Officer is reviewing their LPA Questionnaire task list
+    When the user selects the link 'Are there any health and safety issues on the appeal site?'
+    Then the user is presented with the 'Are there any health and safety issues on the appeal site?' page
+    And the Page title is 'Are there any health and safety issues on the appeal site? - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK'
+
+  Scenario: AC02 - User makes no selection and is provided an error
+    Given user is in the health and safety page
+    When user does not select an option
+    And user selects Save and Continue
+    Then user is shown an error message "Select yes if there are health and safety issues"
+
+  Scenario: AC03 - User selects no and proceeds to task list
+    Given user is in the health and safety page
+    When user selects the option 'No'
+    And user selects Save and Continue
+    Then progress is made to the task list
+    And a Completed status is populated for the task
+
+  Scenario Outline: AC04 - user selects yes and provides further information in text field and proceeds to task list
+    Given user is in the health and safety page
+    And user selects the option 'Yes'
+    And user enters '<health and safety issues>'
+    When user selects Save and Continue
+    Then progress is made to the task list
+    And a Completed status is populated for the task
+    Examples:
+      | health and safety issues |
+      | some health and safety issues |
+      | some more health and safety issues |
+      |  some health and safety issues with a space at the beginning |
+      | health and safety issues with special characters ,. !" }]* & -+%$£@! |
+      | a very long piece of health and safety issues a very long piece of health and safety issues\na very long piece of health and safety issues a very long piece of health and safety issues\na very long piece of health and safety issues a very long piece of health and safety issues\na very long piece of health and safety issues |
+
+  Scenario: AC05 - user does not provides further information in text field and is provided an error
```

Comment:
Perhaps `provides` in this context should be `provide`?

Reply:
Correct, and fixed other feature files.